lms/djangoapps/mobile_api/users/tests.py

```diff
@@ -61,13 +62,14 @@ def test_success(self):
 
 
 @ddt.ddt
-class TestUserEnrollmentApi(UrlResetMixin, MobileAPITestCase, MobileAuthUserTestMixin):
+class TestUserEnrollmentApi(UrlResetMixin, MobileAPITestCase, MobileAuthUserTestMixin, MobileCourseAccessTestMixin):
     """
     Tests for /api/mobile/v0.5/users/<user_name>/course_enrollments/
     """
     REVERSE_INFO = {'name': 'courseenrollment-detail', 'params': ['username']}
     ALLOW_ACCESS_TO_UNRELEASED_COURSE = True
     ALLOW_ACCESS_TO_MILESTONE_COURSE = True
+    ALLOW_ACCESS_TO_NON_VISIBLE_COURSE = True
```

Comment:
Technically seems good; but I could not understand how logically access to non-visible course related to course about